modules/cluster-logging-collector-tolerations.adoc

```diff
@@ -43,27 +47,27 @@ For example:
 $ oc adm taint nodes node1 collector=node:NoExecute
 ----
 +
-This example places a taint on `node1` that has key `collector`, value `node`, and taint effect `NoExecute`.
-You must use the `NoExecute` taint effect. `NoExecute` schedules only pods that match the taint and removes existing pods
-that do not match.
+This example places a taint on `node1` that has key `collector`, value `node`, and taint effect `NoExecute`. You must use the `NoExecute` taint effect. `NoExecute` schedules only pods that match the taint and removes existing pods that do not match.
 
-. Edit the `collection` stanza of the `ClusterLogging` custom resource (CR) to configure a toleration for the logging collector pods:
+. Edit the `collection` stanza of the `ClusterLogging` custom resource (CR) to configure a toleration for the logging collector pods. The following example toleration configuration matches the taint created by the `oc adm taint` command. A pod with this toleration schedules onto `node1`.
 +
+.Example toleration configuration
 [source,yaml]
 ----
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogging
+# ...
+spec:
   collection:
-    logs:
-      type: "fluentd"
-      fluentd:
-        tolerations:
-        - key: "collector"  <1>
-          operator: "Exists"  <2>
-          effect: "NoExecute"  <3>
-          tolerationSeconds: 6000  <4>
+    type: fluentd
```

Comment:
use vector since fluentd is deprecated

Reply:
Thought of doing this but I took it from Calvin's example in the comment of the Jira, which used Fluentd, so I wasn't sure if the config would be the same. Thanks, will update.